README.md

```diff
@@ -40,3 +40,74 @@ if ($file->getSize()) {
 // Access a resource provided by this Divide.IQ server.
 $result = $divideIq->request('stockbase_stock');
 ```
+
+Example Usage orderRequest POST
+=============
```

Comment:
Nitpick, but the idea of using this kind of heading in Markdown is that it looks like a heading commonly used in plaintext files. So the bar of `===` should have the same length as the text above it.